src/syntax_mapping.rs

```diff
@@ -89,8 +89,7 @@ impl<'a> SyntaxMapping<'a> {
         for glob in &[
             "/etc/nginx/**/*.conf",
             "/etc/nginx/sites-*/**/*",
-            "nginx.conf",
```

Comment:
Thanks for your contribution! I see that the duplication was from [the nginx sublime syntax file specifying these under `file_extensions`](https://github.com/brandonwamboldt/sublime-nginx/blob/65f5a63c0d1760c5db2264e50e3b14a7a4567cc0/Syntaxes/nginx.sublime-syntax#L9). But `mime.types` is a full file name, so I am wondering if that is why it was also in this mapping, maybe it behaves differently than expected?

Thus I would feel more confident about this change if we had a `mime.types` file in our syntax tests (https://github.com/sharkdp/bat/tree/master/tests/syntax-tests/source/nginx) to prove that they are still highlighted correctly.

Reply:
I happen to have (a real) one at hand, so I can add this. Good idea!